newa/__init__.py

```diff
@@ -145,13 +181,63 @@ class InitialErratum(Serializable):
 
 @define
 class Erratum(Cloneable, Serializable):
-    """ An eratum """
+    """
+    An eratum
 
+    Represents a set of builds targetting a single release.
+    """
+
+    # TODO: We might need to add more in the future.
     release: str
-    # builds: list[...] = ...
+    builds: list[str]
 
-    def fetch_details(self) -> None:
-        raise NotImplementedError
+    @classmethod
+    def from_errata_tool(cls, erratum_id : str) -> list[Erratum]:
```

Comment:
The input should probably be `Event`. It carries the ET ID, and it's the object that newa code uses to initialize errata structures from. So, when speaking about "erratum ID", the answer should be `event.id`.

Reply:
Right. That sounds reasonable. I'll fix that.